fractalis/config.py

```diff
@@ -43,4 +43,13 @@
 OIDC_SERVER_URL = 'https://keycloak-example.com/auth/realms/transmart-realm/'
 OIDC_CLIENT_ID = 'transmart-client'
 
+DATA_SERVICES = {
+    'data_services': {
+        'test-service': {
```

Comment:
This specific configuration is required for tests to pass. There should be a separate config for functional/unit test purposes, but I did not find a way to properly use another config, when running tests...

Reply:
The trick to change the config in the test code does not work? E.g., here: https://github.com/thehyve/Fractalis/blob/dev/tests/functional/test_data.py#L282

Reply:
unfortunately not, data_services_config is apparently assigned before tests start: https://github.com/thehyve/Fractalis/blob/datasource-whitelist/fractalis/__init__.py#L44 and I tried a few ways to override `data_services_config` in tests, but it did not work.

Reply:
fixed by https://github.com/thehyve/Fractalis/blob/datasource-whitelist/tests/__init__.py

Reply:
Yes, looks great now